eng/devices/catalyst.cake

```diff
@@ -76,7 +76,7 @@ Task("uitest")
 			ArgumentCustomization = args => args
 				.Append($"-f {TARGET_FRAMEWORK}")
 				.Append("-t:Run")
-				.Append("/tl")
+				//.Append("/tl")
```

Comment:
Do we still need these? (and also one below

Reply:
I m going to enable this on net8 branch